addons/html_editor/static/src/main/list/list_plugin.js

```diff
@@ -840,6 +842,7 @@ export class ListPlugin extends Plugin {
     }
 
     handleTab() {
+        this.dependencies.split.splitBlockSegments();
```

Comment:
I'm not a huge fan of the fact that all those random plugins now need to know that they have to call `split.splitBlockSegments()` before doing their stuff ... 

I don't realy have a better solution, from the top of my head ...
I don't know, It just seems wrong...

Maybe we can discuss it bit with the rest of the team to be sure there's no better way to achieve this.

WYT?